packages/ast-tooling/index.ts

```diff
@@ -173,9 +173,42 @@ export function parseJson(content: string) {
     return fleece.evaluate(content);
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function serializeJson(originalInput: string, data: any) {
+export function serializeJson(originalInput: string, data: unknown) {
     // some of the files we need to process contain comments. The default
     // node JSON.parse fails parsing those comments.
-    return fleece.patch(originalInput, data);
+    const spaces = guessIndentString(originalInput);
+    return fleece.stringify(data, { spaces });
```

Comment:
we should only pass `spaces` if the file is already using spaces and not if the file is using tabs

Reply:
oh, nevermind. I see `guessIndentString` returns `undefined` for tabs, so it looks like we're okay there